sklearn/tree/_tree.pyx

```diff
@@ -1123,3 +1123,110 @@ cdef class Tree:
         Py_INCREF(self)
         arr.base = <PyObject*> self
         return arr
+
+
+    def _partial_dependence(self, DTYPE_t[:, ::1] X,
```

Comment:
Should this be a public method of a private class?

Reply:
TBH I don't know what the convention is here.
Some methods of the Tree class are public like `predict` or `apply`, some of them are private like `_add_node`.

Reply:
Is _add_node ever called from outside tree building?

Reply:
oh OK that's the convention. I'll make it public then